Libraries/Animated/Animated.js

```diff
@@ -291,6 +301,13 @@ class DecayAnimation extends Animation {
     window.cancelAnimationFrame(this._animationFrame);
     this.__debouncedOnEnd({finished: false});
   }
+
+  _tryClamping(
```

Comment:
Yeah I agree. Forgot to change the name after changing the function's purpose. :smile: